yoyodyne/models/modules/transformer.py

```diff
@@ -184,6 +220,242 @@ def name(self) -> str:
         return "transformer"
 
 
+class TransformerDecoderSeperateFeatures(nn.TransformerDecoder):
+    """A Transformer decoder with seperate features.
+
+    Adding seperate features into the transformer stack is implemented with
+    TransformerDecoderLayerSeperateFeatures layers.
+    """
+
+    def forward(
+        self,
+        tgt: torch.Tensor,
```

Comment:
For consistency with what we do elsewhere I suggest `target_*` rather than `tgt_`.

Reply:
I think I had forgotten why I had `tgt`, and then I must not have tested before changing it here :( sorry! (see comment about this error).